### DISCUSSION TITLE
moving installed acme script

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [acmesh-official/acme.sh#4838] [General]
I moved the installed acme to a different server same location. However now I am getting

`Register account Error: {"type":"urn:ietf:params:acme:error:malformed","status":400,"detail":"[External Account Binding] The JWS Signature MUST be present"}`

Is there a disadvantage of keeping the same account across multiple servers.

**Comment by f1-outsourcing** [comment]
If I use --test I do not have this error

but weird stuff in the cert

```
1.3.6.1.4.1.11129.2.4.2:
                ......w..l..........9...
."....AP...5..F....\d.......H0F.!......B..?
```

And it is from letsencrypt instead of zerossl

**Comment by neilpang** [comment]
please try to register a new account.
